docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM arti.dev.cray.com/baseos-docker-master-local/alpine:3.12
+FROM docker.io/library/alpine:3.12
```

Comment:
it's kind of dangerous to use docker.io as the source for images. docker.io limits the number of pulls that can happen so this can lead to failures.

Reply:
Though to be honest I think this repo should be deprecated. Container rebuilds should happen from https://github.com/Cray-HPE/container-images and if we need to customize the postgres-operator charts, then we should have a e.g., `postgres-operator-chart` repository that uses the `postgres-operator` and `postgres-operator-ui` charts as dependencies.

All that said, I wouldn't think too hard about this. Just use `artifactory.algol60.net/docker.io/library/alpine` and be done with it.

Reply:
Thank you for your comments! I'll make the change to use our mirror.